### DISCUSSION TITLE
Any thoughts on porting over to OpenXR now that its the new standard

### DISCUSSION THREAD

**Discussion by ImagineTheGames** [Jordonbc/VREngine#12] [Ideas]
OpenXR being the new standard, do you have any thoughts/plans on this migration?

**Comment by Jordonbc** [comment]
I do agree that VREngine should support this but I have experimented in the past with getting OpenXR working when it was first added to Unreal Engine 4.23 but was unable to move over because SteamVR didn't fully implement OpenXR.

It looks like its now fully supported so I will start migrating over in a separate branch.
